presto-main/src/test/java/com/facebook/presto/execution/TestQueryStats.java

```diff
@@ -57,10 +57,11 @@
                     succinctBytes(116L),
                     117L,
                     succinctBytes(118L),
-                    new Duration(119, NANOSECONDS),
```

Comment:
Changing these unrelated values makes the diff hard to read. Use a new number for the new value rather than trying to keep the values sequential. The only change here should be adding the new value.